Tools/_lust/replace_nt.py

```
@@ -7,17 +7,36 @@
+
 def process_ftl_files(locale_dir: Path):
     if not locale_dir.exists():
         print(f"Директория {locale_dir} не найдена, создаём её")
         locale_dir.mkdir(parents=True, exist_ok=True)
 
     for file_path in locale_dir.rglob("*.ftl"):
         content = file_path.read_text(encoding="utf-8")
-        for pattern, replacement in REPLACEMENTS.items():
-            content = re.sub(pattern, replacement, content)
+        lines = []
+        for line in content.splitlines(True):
```

Ruff (0.12.2)

Boolean positional value in function call

(FBT003)